plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -1479,7 +1479,6 @@ public void finishStatisticsCollection(ConnectorSession session, ConnectorTableH
                     .collect(toImmutableMap(HiveColumnHandle::getName, column -> ImmutableSet.copyOf(metastore.getSupportedColumnStatistics(column.getType()))));
             Supplier<PartitionStatistics> emptyPartitionStatistics = Suppliers.memoize(() -> createEmptyPartitionStatistics(columnTypes, columnStatisticTypes));
 
-            int usedComputedStatistics = 0;
             List<Type> partitionTypes = handle.getPartitionColumns().stream()
```

Comment:
The PR is not accompanied by corresponding tests because I don't see a way to consistently simulating a situation where partitions get effectively removed during the computation of statistics.